### PR TITLE
 feature/12-monitor-running-in-windows-local-system-account

### PR DESCRIPTION
**Pull request by jhorcicka**
Solves #12 

Chapter 13, monitor, "Creating a resources folder on Windows" subsection was added together with "Example with a missing systemprofile resources folder".

doc-update

### CODE REVIEW

File: src/docbkx/chapter13-getting-started-monitor.xml [hunk 1]
```diff
@@ -30,14 +30,65 @@
...
 			suggestions for login credentials directly on the screen, to get you
 			started quickly.
 		</para>
+		<section>
+			<title>Creating a resources folder on Windows</title>
+			<para>
+				DataCleaner monitor uses a special resources folder. Typically it is created automatically during 
+				installation as <emphasis>C:\Windows\Users\USER_NAME\.datacleaner</emphasis>. However, when using 
+				Apache Tomcat and its system service then this resources folder needs to be created manually as 
+				<emphasis>C:\Windows\System32\config\systemprofile\.datacleaner</emphasis>.
```
Comment (LosD) [inline src/docbkx/chapter13-getting-started-monitor.xml]: Windows Explorer wont let you create any `.folder` names, so they need to do it from the command line. We'll probably need to explain how.

Comment (LosD) [inline src/docbkx/chapter13-getting-started-monitor.xml]: D'oh, explained below. Nevermind me :)

File: src/docbkx/chapter13-getting-started-monitor.xml [hunk 2]
```diff
@@ -30,14 +30,65 @@
...
+			<para>
+				Open your browser at <link xl:href="http://localhost:8080/DataCleaner-monitor/scheduling">
+				http://localhost:8080/DataCleaner-monitor/scheduling</link>. 
+				<mediaobject>
+					<imageobject>
+						<imagedata fileref="monitor_01_jobs.png" format="PNG" scalefit="1" />
+					</imageobject>
+				</mediaobject>
+				When you try to run, let's say, 'Customer duplicates' job in DataCleaner monitor, you get following 
+				exception (missing valid license).
```
Comment (LosD) [inline src/docbkx/chapter13-getting-started-monitor.xml]: It's weird. When I try without the license, I don't even get this far. I get a 404 immediately when logging in.

Comment (LosD) [inline src/docbkx/chapter13-getting-started-monitor.xml]: Might be because the default is the dashboard page?

Comment (jhorcicka) [inline src/docbkx/chapter13-getting-started-monitor.xml]: What do you mean by that "default is dashboard"? I can view all the pages without license and even run "simple" jobs (probably those that do not require remote services).

Comment (LosD) [inline src/docbkx/chapter13-getting-started-monitor.xml]: I just meant that if you open the monitor without specifying anything but "http://localhost:8080/DataCleaner-monitor/" then it'll go to the dashboard after logging in by default.

It's weird that you can do anything. I'm completely locked out without a license.

Comment (jhorcicka) [inline src/docbkx/chapter13-getting-started-monitor.xml]: Can that be caused by version? I have 5.0.3 enterprise.

**Comment by jhorcicka** [comment]
Example part was replaced by a general statement (Problems? Create folder!). 

docs

File: src/docbkx/chapter13-getting-started-monitor.xml [hunk 3]
```diff
@@ -30,14 +30,45 @@
...
+				installation as<emphasis>C:\Windows\Users\USER_NAME\.datacleaner</emphasis>. However, when using
+				Apache Tomcat and its system service then this resources folder needs to be created manually as
+				<emphasis>C:\Windows\System32\config\systemprofile\.datacleaner</emphasis>. If you encounter any
+				problems while using the system service, make sure the resource folder is available.
+			</para>
+		</section>
+		<section>
+			<title>Manual creation of Windows systemprofile resource folder</title>
+			<para>
+				Default Windows explorer does not allow you to create a folder starting with a dot ('.')
```
Comment (LosD) [inline src/docbkx/chapter13-getting-started-monitor.xml]: I think Windows Explorer is usually written with capital E. Maybe drop the "Default" then, since it's then clear you are talking about the built-in one.

File: src/docbkx/chapter13-getting-started-monitor.xml [hunk 4]
```diff
@@ -30,14 +30,45 @@
...
+				so use another file manager or command prompt: 
+				<emphasis>mkdir C:\Windows\System32\config\systemprofile\.datacleaner</emphasis>
```
Comment (LosD) [inline src/docbkx/chapter13-getting-started-monitor.xml:56]: I guess this needs to be an elevated (Administrator) command prompt?

Comment (jhorcicka) [inline src/docbkx/chapter13-getting-started-monitor.xml:56]: Yes.

File: src/docbkx/chapter13-getting-started-monitor.xml [hunk 5]
```diff
@@ -30,14 +30,45 @@
...
 		</mediaobject>
 		<para>In the community edition of DataCleaner, you will find
 			suggestions for login credentials directly on the screen, to get you
 			started quickly.
 		</para>
+		<section>
+			<title>Creating a resources folder on Windows</title>
+			<para>
+				DataCleaner monitor uses a special resources folder. Typically it is created automatically during
+				installation as<emphasis>C:\Windows\Users\USER_NAME\.datacleaner</emphasis>. However, when using
```
Comment (LosD) [inline src/docbkx/chapter13-getting-started-monitor.xml]: Isn't it C:\Users\USER_NAME?

And I only ever use English Windows, so I'm not sure, but I guess they still translate folder names for localized Windows versions? Maybe rephrase into something like "A `.datacleaner` folder is created in your user home by the installer,  in an English Windows 7 this would probably be `C:\Users\YOUR_USER_NAME\.datacleaner`"

File: src/docbkx/chapter13-getting-started-monitor.xml [hunk 6]
```diff
@@ -30,14 +30,45 @@
...
+				Apache Tomcat and its system service then this resources folder needs to be created manually as
+				<emphasis>C:\Windows\System32\config\systemprofile\.datacleaner</emphasis>. If you encounter any
+				problems while using the system service, make sure the resource folder is available.
+			</para>
+		</section>
+		<section>
+			<title>Manual creation of Windows systemprofile resource folder</title>
+			<para>
+				Windows Explorer does not allow you to create a folder starting with a dot ('.')
```
Comment (jhorcicka) [inline src/docbkx/chapter13-getting-started-monitor.xml:54]: Windows Explorer.

File: src/docbkx/chapter13-getting-started-monitor.xml [hunk 7]
```diff
@@ -30,14 +30,45 @@
...
+				so use another file manager or command prompt (in Administrator mode):
```
Comment (jhorcicka) [inline src/docbkx/chapter13-getting-started-monitor.xml:55]: Administrator mode.

**Comment by LosD** [comment]
Except maybe a possible rephrasing of the default .datacleaner folder, LGTM.

File: src/docbkx/chapter13-getting-started-monitor.xml [hunk 8]
```diff
@@ -30,14 +30,46 @@
...
 		<para>In the community edition of DataCleaner, you will find
 			suggestions for login credentials directly on the screen, to get you
 			started quickly.
 		</para>
+		<section>
+			<title>Creating a resources folder on Windows</title>
+			<para>
+				DataCleaner monitor uses a special resources folder. Typically it is created automatically during
+				installation. In an English Windows 7 this would probably be 
+				<emphasis>C:\Users\YOUR_USER_NAME\.datacleaner</emphasis>. However, when using
```
Comment (jhorcicka) [inline src/docbkx/chapter13-getting-started-monitor.xml:45]: "Localization".

**Comment by LosD** [comment]
LGTM!
